Classes/Controller/BibliografieController.php

```diff
@@ -219,7 +219,7 @@ function request($url, $requestCount=0)
             $data = array_merge($data, $dataNext);
         }
 
-        if ( $this->settings['debug'] == true ) \TYPO3\CMS\Core\Utility\DebugUtility::debug( $collections, 'Debug: ' . __FILE__ . ' in Line: ' . __LINE__ . ' Function: '. __FUNCTION__);
+        if ( $this->settings['debug'] == true ) \TYPO3\CMS\Core\Utility\DebugUtility::debug( $data, 'Debug: ' . __FILE__ . ' in Line: ' . __LINE__ . ' Function: '. __FUNCTION__);
```

Comment:
Please follow [PSR-2](https://www.php-fig.org/psr/psr-2/) if you change anything.

```suggestion
        if ($this->settings['debug'] == true) {
            \TYPO3\CMS\Core\Utility\DebugUtility::debug($data, 'Debug: ' . __FILE__ . ' in Line: ' . __LINE__ . ' Function: '. __FUNCTION__);
        }
```